content/en/monitors/guide/how-to-set-up-rbac-for-monitors.md

```diff
@@ -0,0 +1,216 @@
+* At the bottom of the form, specify which roles, in addition to the creator, are allowed to edit the monitor.
+
+{{< img src="/monitors/guide/monitor_rbac_restricted.jpg" alt="RBAC Restricted Monitor"  >}}
+
+For more information, see [Monitors Permissions][1].
+
+[1]: /monitors/notify/#permissions
+{{% /tab %}}
+
+{{% tab "API" %}}
+
+Use the [List Roles API endpoint][1] to get the list of roles and their id.
+
+```bash
+curl --request GET 'https://api.datadoghq.com/api/v2/roles' \
+--header 'DD-API-KEY: <DD-API-KEY>' \
+--header 'DD-APPLICATION-KEY: <DD-APPLICATION-KEY>'
+```
+
+```bash
+{
+    "meta": {
+        "page": {
+            "total_filtered_count": 4,
+            "total_count": 4
+        }
+    },
+    "data": [
+        {
+            "type": "roles",
+            "id": "89f5dh86-e470-11f8-e26f-4h656a27d9cc",
+            "attributes": {
+                "name": "Corp IT Eng - User Onboarding",
+                "created_at": "2018-11-05T21:19:54.105604+00:00",
+                "modified_at": "2018-11-05T21:19:54.105604+00:00",
+                "user_count": 4
+            },
+            "relationships": {
+                "permissions": {
+                    "data": [
+                        {
+                            "type": "permissions",
+                            "id": "984d2rt4-d5b4-13e8-a5yf-a7f560d33029"
+                        },
+                        ...
+                    ]
+                }
+            }
+        },
+        ...
+    ]
+}
+```
+
+Use the [Create][2] or [Edit a monitor][3] API endpoint and the `restricted_roles` parameter to restrict monitor editing to a specific set of roles and to the monitor's creator. 
+
+**Note:** You can specify one or multiple role UUIDs. Setting `restricted_roles` to `null` allows monitor editing for all users with [Monitor Write permissions][4].
+
+```bash
+curl --location --request POST 'https://api.datadoghq.com/api/v1/monitor' \
+--header 'Content-Type: application/json' \
+--header 'DD-API-KEY: <DD-API-KEY>' \
+--header 'DD-APPLICATION-KEY: <DD-APPLICATION-KEY>' \
+--data-raw '{
+  "message": "You may need to add web hosts if this is consistently high.",
+  "name": "Bytes received on host0",
+  "options": {
+    "no_data_timeframe": 20,
+    "notify_no_data": true
+  },
+  "query": "avg(last_5m):sum:system.net.bytes_rcvd{host:host0} \u003e 100",
+  "tags": [
+    "app:webserver",
+    "frontend"
+  ],
+  "type": "query alert",
+  "restricted_roles": ["89f5dh86-e470-11f8-e26f-4h656a27d9cc"]
+}'
+```
+
+For more information, see [Roles][5] and [Monitors API Reference][6] .
+
+
+[1]: /api/latest/roles/#list-roles
+[2]: /api/latest/monitors/#create-a-monitor
+[3]: /api/latest/monitors/#edit-a-monitor
+[4]: /account_management/rbac/permissions/#monitors
+[5]: /api/latest/roles/
+[6]: /api/latest/monitors/
+{{% /tab %}}
+{{< /tabs >}}
+
+## Migrate locked monitors to use RBAC
```

Comment:
Actually that title does not really work since we also cover non locked monitors in the UI section.

Note to self before closing this: make sure to update any link that was referring to this.